compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_cores.mlir

```diff
@@ -300,3 +300,28 @@ module {
     return
   }
 }
+
+// -----
+
+// CHECK-LABEL: @insert_truncf_within_core
+//       CHECK:     scf.forall
+//       CHECK:       amdaie.tile
+//       CHECK:       amdaie.core
+//       CHECK:         vector.transfer_read
+//       CHECK:         arith.truncf
+//       CHECK:         vector.transfer_write
+//       CHECK:         amdaie.end
```

Comment:
I prefer that we don't indent the `CHECK` further than one space from `//` to be consistent with the rest of the codebase (and this file).